playground/playground.js

```diff
@@ -1,9 +1,53 @@
-import init, {transform} from '../node/pkg';
-
 let enc = new TextEncoder();
 let dec = new TextDecoder();
 let inputs = document.querySelectorAll('input[type=number]');
 
+async function loadVersions() {
+  const {versions} = await fetch('https://data.jsdelivr.com/v1/package/npm/@parcel/css-wasm').then(r => r.json());
+  versions
+    .map(v => {
+      const option = document.createElement('option');
+      option.value = v;
+      option.textContent = v;
+      return option;
+    })
+    .forEach(o => {
+      version.appendChild(o);
+    })
+}
+
+async function loadWasm() {
+  function load(url) {
+    return new Promise((resolve, reject) => {
+      const cleanup = (script) => {
+        URL.revokeObjectURL(script.src);
+        script.remove();
+      };
+      const moduleBlob = new Blob([
+        `import * as m from '${url}';`,
+        `window.$cssWasm = m;`,
+      ], {type: 'text/javascript'});
+      const script = Object.assign(document.createElement('script'), {
+        type: 'module',
+        src: URL.createObjectURL(moduleBlob),
+        onerror() {
+          reject(new Error(`Failed to import: ${url}`));
+          cleanup(script);
+        },
+        onload() {
+          resolve();
+          cleanup(script);
+        },
+      });
+
+      document.head.appendChild(script);   
+    })
+  }
+  
+  await load(`https://cdn.jsdelivr.net/npm/@parcel/css-wasm@${version.value}/parcel_css_node.js`);
```

Comment:
I think we could probably use dynamic import for this. Looks like Parcel doesn't like absolute URLs there yet, but you could do something like this:

```js
let wasm = await new Function('version', 'return import(`https://cdn.jsdelivr.net/npm/@parcel/css-wasm@${version}/parcel_css_node.js`)')(version.value);
await wasm.default();
return wasm;
```

And then pass the returned wasm object to the `update` function, rather than having a global variable?

Reply:
Yes, I was using dynamic import for this, but I didn't find how to dynamic import an external url resource in Parcel, so I use this for a kind of fallback.
Will switch to the `new Function` way, it seems more convenient and readable, but the `update` function is used to handle `oninput` and `onchange` event, passing the wasm object is not handy for that, so I'm leaving a `let` variable to hold the wasm object to keep it simple~